main.go

```diff
@@ -25,6 +23,18 @@ func main() {
 	app.Version = VERSION
 	app.Usage = "An external resource based scheduler for Rancher."
 	app.Action = run
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "metadata-address",
+			Usage: "The metadata service address",
+			Value: "rancher-metadata",
+		},
+		cli.StringFlag{
+			Name:  "listen",
+			Usage: "Listen on this port for healthchecks",
+			Value: ":80",
```

Comment:
I did this so we could listen on a specific interface, instead of all interfaces. Just my $.02, makes the param more flexible.